k8s/components/service/resources/deployment.yaml

```diff
@@ -38,6 +38,8 @@ spec:
           name: sbomer
           imagePullPolicy: IfNotPresent
           env:
+            - name: SBOMER_SBOM_SBOM_DIR
```

Comment:
Shouldn't it be `SBOMER_SBOM_DIR` ? See chenges in file `service/src/main/resources/application.yaml` below
```
  sbom:
    sbom-dir: "${SBOMER_SBOM_DIR}"
```